docs/dev.md

```diff
@@ -67,19 +67,35 @@ Integration tests can read the following system properties to obtain information
 
 ##### devc, Container Mode
 
-This is a technology preview.
+This is a technology preview. The features and parameters described below may change in future milestones or releases of the Liberty Maven plugin.
 
-Start a Liberty server in a local container using the Docker file that you provide. The Docker file must copy the application .war file and the server configuration files that the application requires into the container. An alternative to the `devc` goal is to specify the `dev` goal with the `-Dcontainer` option. A sample Docker file is shown in [Building an application image](https://github.com/openliberty/ci.docker/#building-an-application-image). Note that the context of the `docker build` command used to generate the container image is the directory containing the docker file.
+Start a Liberty server in a local container using the Dockerfile that you provide. An alternative to the `devc` goal is to specify the `dev` goal with the `-Dcontainer` option. 
+
+The Dockerfile must copy the application .war file and the server configuration files that the application requires into the container. A sample Dockerfile is shown in [Building an application image](https://github.com/openliberty/ci.docker/#building-an-application-image). Note that the context of the `docker build` command used to generate the container image is the directory containing the Dockerfile. When dev mode exits the container is stopped and deleted and the logs are preserved as described below.
 
 You need to install the Docker runtime (e.g. Docker Desktop) locally to use this Maven goal. You can examine the commands used to build and run the container by viewing the console output of dev mode.
 
-When dev mode runs in a container it still provides the same features. It monitors files for changes and runs tests either automatically or on command. This mode also allows you to attach a debugger to work on your application. You can review the logs generated by your server in the Liberty directory in your project e.g. target/liberty/wlp/usr/servers/defaultServer/logs.
+When dev mode runs in a container it still provides the same features. It monitors files for changes and runs tests either automatically or on demand. This mode also allows you to attach a debugger to work on your application. You can review the logs generated by your server in the Liberty directory in your project e.g. target/liberty/wlp/usr/servers/defaultServer/logs.
```

Comment:
`When dev mode runs in a container` is inaccurate since only the server is in a container.  Perhaps use one of the following:
- `When dev mode runs with the server in a container, `
- `When dev mode runs with container support, `
- `When dev mode runs in container mode, ` (although this has too many "modes")

Reply:
I knew I was playing fast and loose here. I like "container support" to avoid being too repetitive.